### PR TITLE
add parameter --skip_variant_calling

### PR DESCRIPTION
**Pull request by Lucpen**
## PR checklist

- [x] add parameter --skip_variant_calling to speed things up if you want to build a database with drop
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/genomic-medicine-sweden/tomte/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core pipelines lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test.config [hunk 1]
```diff
@@ -26,6 +26,7 @@ params {
     // Skip when GITHUB actions
     skip_drop_ae = System.getenv("GITHUB_ACTIONS").equals(null) ? false : true
     skip_drop_as = System.getenv("GITHUB_ACTIONS").equals(null) ? false : true
+    skip_variant_calling = true
```
Comment (Lucpen) [inline conf/test.config:29]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,20 +8,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### `Added`
 
 - Functionality to create DROP databases and to add samples to existing ones [#147](https://github.com/genomic-medicine-sweden/tomte/pull/147)
+- A switch `--skip_variant_calling` for variant calling [#168](https://github.com/genomic-medicine-sweden/tomte/pull/168)
```
Comment (Lucpen) [inline CHANGELOG.md:11]:
```suggestion
- A switch `--skip_variant_calling` for variant calling [#169](https://github.com/genomic-medicine-sweden/tomte/pull/169)
```

File: nextflow_schema.json [hunk 3]
```diff
@@ -273,6 +273,12 @@
             "default": "",
             "fa_icon": "fas fa-filter",
             "properties": {
+                "skip_variant_calling": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Skip variant calling for all samples.",
+                    "fa_icon": "fas fa-toggle-off"
+                },
```
Comment (fellen31) [inline nextflow_schema.json:281]: Perhaps have these together with the other skips below?

**Review by fellen31 — APPROVED** [review]
Just one question, otherwise LGTM! 👍

File: nextflow_schema.json (hunk 3 above)
Comment (Lucpen) [reply nextflow_schema.json]: Done! Thanks for the super fast review 😄

**Pull request merged**